.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,45 @@
+name: Bug Report
+description: Something is broken? Create a report to help us improve.
+title: "[Bug]: "
+labels: [bug]
+body:
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: Describe what happens and the steps to recreate the problem. Issues that we can't reproduced will be closed.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Video/Screenshots
+      description: Please provide a screenshot or a short video presenting the issue
+    validations:
+      required: true
+  - type: input
+    id: demo
+    attributes:
+      label: Provide a link to the demo with the bug reproduction
+      description: Use StackBlitz, CodeSandbox, CodePen, JSFiddle, or any other online sandbox.
```

IMO this should be required. It's really hard for us to fix a bug without a demo.

```suggestion
      description: Use StackBlitz, CodeSandbox, CodePen, JSFiddle, or any other online sandbox.
    validations:
      required: true
```